CHANGELOG.md

```diff
@@ -6,11 +6,13 @@ CHANGELOG
 
 **ENHANCEMENTS**
 
+- Add support for CentOS 8
```

Comment:
shall we add: 'in all Commercial and GovCloud regions'?

Reply:
probably just in global regions (commercial)

Reply:
Fixed